### PR TITLE
Feat/device image endpoint

### PR DESCRIPTION
**Pull request by acosferreira**

### CODE REVIEW

**Review by biancarosa — COMMENTED** [review]
Can we add some tests for this one?

File: pkg/routes/devices.go [hunk 1]
```diff
@@ -103,3 +104,24 @@ func GetUpdateAvailableForDevice(w http.ResponseWriter, r *http.Request) {
...
+	uuid := chi.URLParam(r, "DeviceUUID")
+
+	client := inventory.InitClient(r.Context())
+	var device inventory.InventoryResponse
+	device, err := client.ReturnDevicesByID(uuid)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+	}
+
+	currentCheckSum := device.Result[len(device.Result)-1].Ostree.RpmOstreeDeployments[len(device.Result[len(device.Result)-1].Ostree.RpmOstreeDeployments)-1].Checksum
```
Comment (maxamillion) [inline pkg/routes/devices.go:119]: Do we need to make sure `len(device.Result)` isn't zero before doing this to make sure we don't index out of bounds?

Comment (acosferreira) [reply pkg/routes/devices.go:119]: I've added this validation in another pr #241

**Comment by acosferreira** [comment]
Closing due to rebase and new branch needed

**Pull request closed without merging**